From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexander Khabarov <alexander.khabarov@arm.com>
Date: Fri, 20 Oct 2023 10:57:18 +0000
Subject: [PATCH] Correct builtin headers prefix

`$${src#*clang/lib/Headers}` works when building from the
`llvm-project` root as well.
---
 clang/BUILD.bazel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clang/BUILD.bazel b/clang/BUILD.bazel
index 037719a51..ec87a37e2 100644
--- a/clang/BUILD.bazel
+++ b/clang/BUILD.bazel
@@ -1691,7 +1691,7 @@ genrule(
     outs = [hdr.replace("lib/Headers/", "staging/include/") for hdr in builtin_headers],
     cmd = """
        for src in $(SRCS); do
-         relsrc=$${src#*"$(WORKSPACE_ROOT)"/clang/lib/Headers}
+         relsrc=$${src#*clang/lib/Headers}
          target=$(@D)/staging/include/$$relsrc
          mkdir -p $$(dirname $$target)
          cp $$src $$target
